isic_kredit/cs.lproj/Localizable.strings

```diff
@@ -0,0 +1,18 @@
+/* 
+  Localizable.strings
+  isic_kredit
+
+  Created by Petr Dusek on 9/02/2019.
+  Copyright © 2019 Petr Dusek. All rights reserved.
+*/
+
+"Username"  = "Uživatelské jméno";
+"Password" = "Heslo";
+"Login" = "Přihlásit se";
+"User" = "Uživatel";
+"Account balance" = "Na účtu máte";
```

Comment:
Takovýhle klíče jsou zlo, ale to neřeš, tomu jsme se na cvikách moc nevěnovali. Vysvětlím klidně při odevzdání, kdyžtak se připomeň.